examples/training/1-Intro to Bricks.ipynb

```diff
@@ -0,0 +1,741 @@
+      "\n",
+      "January 29, 2023\n",
+      "\n",
+      "But given the fierce rivalry between the Eagles and the New York Giants, who the Super Bowl-bound team had comfortably defeated in the previous round of the NFL Playoffs, many were left questioning the move.\n",
+      "\n",
+      "Did y'all lose a bet, ESPN contributor Mina Kimes asked in response to the tweet, while Giants running back Matt Breida also expressed his disbelief.\n",
+      "\n",
+      "SMHð¤¦ð¾âï¸— Matt Breida (@MattBreida)\n",
+      "\n",
+      "January 30, 2023\n",
+      "\n",
+      "As the representative for the Empire State Building, and a diehard Giants fan, let me be on the record saying that this is absolutely ridiculous, said New York City councilman Keith Powers.\n",
+      "\n",
+      "The Giants' Twitter account also acknowledged the divisive decision, writing: I'm just here for the comments.\n",
+      "\n",
+      "The Empire State Building, whose original tweet honoring the Eagles was viewed nearly 30 million at the time of writing, said the color switch hurt us more than it hurt you — but only after mocking another tweet calling the New York landmark lame.\n",
+      "\n",
+      "The building was later lit in red to celebrate the Kansas City Chiefs' AFC Championship win against the Cincinnati Bengals.\n",
+      "\n",
+      "In Philadelphia, meanwhile, Eagles fans poured onto the streets on Sunday night. Large crowds gathered in the city as people climbed up light posts, street signs, and on top of a bus stop canopy.\n",
+      "\n",
+      "The city announced street closures and vehicle restrictions in Philadelphia's city center due to Eagles celebratory activity between 8th to 20th streets and Race to Lombard streets, the city's Office of Emergency Management tweeted on Sunday night.\n",
+      "\n",
+      "Philadelphians, let's celebrate joyously, safely, and respectfully and show the same love we have for our team to our city. Go Birds! Mayor Jim Kenney tweeted.\n",
+      "\n",
+      "The Eagles and the Chiefs face off in Super Bowl LVII on February 12.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"\\n\\n\".join([str(el) for el in elements]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e51c26ed",
+   "metadata": {},
+   "source": [
+    "## Cleaning bricks <a id=\"cleaning\"></a>\n",
+    "\n",
+    "As part of data preparation for NLP model, it's common to need to clean up your data prior to passing it into a model. If there's unwanted content in your output, it could impact the quality of your NLP model. To help with this, the `unstructured` library includes cleaning bricks to help users sanitize output before sending it to downstream applications. You can check out our [documentation](https://unstructured-io.github.io/unstructured/bricks.html#cleaning) for a full list of cleaning bricks.\n",
```

Comment:
```suggestion
    "As part of data preparation for an NLP model, it's common to need to clean up your data prior to passing it into the model. If there's unwanted content in your output, it could impact the quality of your NLP model. To help with this, the `unstructured` library includes cleaning bricks to help users sanitize output before sending it to downstream applications. You can check out our [documentation](https://unstructured-io.github.io/unstructured/bricks.html#cleaning) for a full list of cleaning bricks.\n",
```